src/form/style/index.less

```diff
@@ -329,7 +333,6 @@ form {
 }
 
 @media (max-width: @screen-xs-max) {
-  .make-vertical-layout();
```

Comment:
Cannot be deleted, when screen less than xs, label align not correct, ref https://github.com/ant-design/ant-design/issues/20834

Reply:
hmm will close then. there is a weird bug that the compatible module will break the new form component styles. need more investigation

Reply:
yep. you can report a new issue or code example if you still have